src/gluonts/evaluation/utils.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from pathlib import Path
+from typing import Iterator, Tuple
+import logging
+import numpy as np
+import pandas as pd
+
+from gluonts.dataset.common import DataEntry, Dataset, ListDataset
+from gluonts.dataset.jsonl import JsonLinesFile
+from gluonts.evaluation import Evaluator
+from gluonts.model.forecast import Forecast, SampleForecast
+from gluonts.dataset.util import forecast_start
+
+logger = logging.getLogger(__name__)
+
+
+def load_forecasts_from_json_file(path: Path) -> list:
+    ds = []
+    for line in JsonLinesFile(path):
+        ds.append(line.content)
+
+    return ds
+
+
+def numpify_samples(fcsts: list) -> list:
+    for item in fcsts:
+        item["SageMakerOutput"]["samples"] = np.array(
```

Comment:
That seems quite specific (in multiple ways).

1. Why is this a `SageMakerOutput`? What about locally run forecasts? 

2. This assumes that the user would have saved the samples, which I think most users don't do, actually, because it is more efficient to save the quantiles. If quantiles are present, we could load this as a `QuantileForecast` instead and then run the evaluation. This way, we could accomodate all use cases.